apps/db_dump/Makefile

```diff
@@ -6,24 +6,22 @@ CFLAGS_FINAL=${CFLAGS} ${BROKER_CFLAGS} ${BROKER_CPPFLAGS} -I${R}/include -I${R}
 OBJS = \
 	   db_dump.o \
 	   print.o \
-	   stubs.o
-
-BROKER_OBJS = \
-	   ${R}/src/memory_mosq.o \
-	   ${R}/src/memory_public.o \
-	   ${R}/src/packet_datatypes.o \
-	   ${R}/src/persist_read.o \
-	   ${R}/src/persist_read_v234.o \
-	   ${R}/src/persist_read_v5.o \
-	   ${R}/src/property_mosq.o \
-	   ${R}/src/topic_tok.o \
-	   ${R}/src/utf8_mosq.o
+	   stubs.o \
+	   memory_mosq.o \
+	   memory_public.o \
+	   packet_datatypes.o \
+	   persist_read.o \
+	   persist_read_v234.o \
+	   persist_read_v5.o \
+	   property_mosq.o \
+	   topic_tok.o \
+	   utf8_mosq.o
 
 .PHONY: all clean reallyclean
 
 all : mosquitto_db_dump
 
-mosquitto_db_dump : ${OBJS} ${BROKER_OBJS}
+mosquitto_db_dump : ${OBJS}
```

Comment:
This also should remain as it was, and the related changes further down reverted.

Reply:
Ok. Simply dropped this commit.